contracts/test/vault/harvester.fork-test.js

```diff
@@ -14,22 +18,122 @@ describe("ForkTest: Harvester", function () {
+      isOethFixture: true,
+    });
+    await hotDeployOption(fixture, null, {
+      isOethFixture: false,
+    });
+  });
+
+  describe("with Curve", () => {
+    it("Should swap CRV for WETH", async () => {
+      const {
+        oethHarvester,
+        strategist,
+        convexEthMetaStrategy,
+        oethDripper,
+        crv,
+        weth,
+      } = fixture;
+      const wethBefore = await weth.balanceOf(oethDripper.address);
+
+      // Send some rewards to the strategy
+      await setERC20TokenBalance(convexEthMetaStrategy.address, crv, "1000");
+
+      // prettier-ignore
+      const tx = await oethHarvester
+        .connect(strategist)["harvestAndSwap(address)"](convexEthMetaStrategy.address);
+
+      await expect(tx).to.emit(convexEthMetaStrategy, "RewardTokenCollected");
+      await expect(tx).to.emit(oethHarvester, "RewardTokenSwapped");
+      await expect(tx).to.emit(oethHarvester, "RewardProceedsTransferred");
+
+      // Should've transferred swapped WETH to Dripper
+      expect(await weth.balanceOf(oethDripper.address)).to.be.gt(wethBefore);
+    });
+  });
+
+  // Commenting this out since none of the strategies have anything
+  // that we want to harvest on Uniswap V2 or V3 right now.
+
+  // describe("with Uniswap V3", () => {
+  //   it("Should swap CRV for USDT", async () => {
+  //     const { harvester, timelock, crv } = fixture;
+  //     const crvBefore = await crv.balanceOf(harvester.address);
+
+  //     await harvester.connect(timelock).swapRewardToken(crv.address);
+
+  //     expect(await crv.balanceOf(harvester.address)).to.equal(
+  //       crvBefore.sub(oethUnits("4000"))
+  //     );
+  //   });
+  // });
+
+  describe("with Balancer", () => {
+    it("Should swap BAL and AURA for WETH", async () => {
+      const {
+        oethHarvester,
+        strategist,
+        bal,
+        aura,
+        weth,
+        oethDripper,
+        balancerREthStrategy,
+      } = fixture;
+
+      const wethBefore = await weth.balanceOf(oethDripper.address);
+
+      // Send some rewards to the strategy
+      await setERC20TokenBalance(balancerREthStrategy.address, bal, "1000");
+      await setERC20TokenBalance(balancerREthStrategy.address, aura, "1000");
+
+      // prettier-ignore
+      const tx = await oethHarvester
+        .connect(strategist)["harvestAndSwap(address)"](balancerREthStrategy.address);
+
+      await expect(tx).to.emit(balancerREthStrategy, "RewardTokenCollected");
+      await expect(tx).to.emit(oethHarvester, "RewardTokenSwapped");
+      await expect(tx).to.emit(oethHarvester, "RewardProceedsTransferred");
+
+      // Should've transferred everything to Harvester
+      expect(await bal.balanceOf(balancerREthStrategy.address)).to.equal("0");
+      expect(await aura.balanceOf(balancerREthStrategy.address)).to.equal("0");
+
+      // Should've transferred swapped WETH to Dripper
+      expect(await weth.balanceOf(oethDripper.address)).to.be.gte(wethBefore);
```

Comment:
This line should pass even when no WETH has been collected (because of gt**e**  -> making it greater or equal) 

Lets check for balance being greater than some margin: 
```
expect(await weth.balanceOf(oethDripper.address)).to.be.gte(wethBefore.add(parseUnits("0.2", 18)));
```